lib/sync/installation.js

```diff
@@ -146,7 +146,11 @@ module.exports.processInstallation = (app, queues) => {
             })
           }
 
-          throw new Error(`Failed Jira API request: ${err.response.status}`)
+          if (err.response && err.response.status) {
+            throw new Error(`Failed Jira API request: ${err.response.status}`)
```

Comment:
I wonder if we should log the original exception here. Otherwise, we would only keep the status and all error details are inaccessible for debugging.